coc/http.py

```diff
@@ -464,12 +471,16 @@ async def initialise_keys(self):
             LOG.info("Retrieved %s valid keys from the developer site.", len(self._keys))
 
             if len(self._keys) < self.key_count:
-                for key in (k for k in keys if k["name"] == self.key_names and ip not in k["cidrRanges"]):
+                for key in keys:
+                    if key["name"] != self.key_names or ip in key["cidrRanges"]:
+                        continue
                     LOG.info(
                             "Deleting key with the name %s and IP %s (not matching our current IP address).",
                             self.key_names, key["cidrRanges"],
                     )
-                    await session.post("https://developer.clashofclans.com/api/apikey/revoke", json={"id": key["id"]})
+                    resp = await session.post("https://developer.clashofclans.com/api/apikey/revoke", json={"id": key["id"]})
```

Comment:
URL should be a constant

Reply:
all of that is old code, we can do a complete code review for release 2.4 together with the docs rework

Reply:
Roger